File: public/phase-2-20160620022203/JAVA/elegance-of-a-sequence.java
// Name : Elegance of a Sequence
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : length of sequence of a
		// a : sequence a
		int index;
		Scanner in = new Scanner(System.in);
		int n = Integer.parseInt(in.nextLine());
		int[] a = new int[n];
		String[] a_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			a[index] = Integer.parseInt(a_elements[index]);


		// write your code here
		// store your results in `k`,`b`

		// output
		// Dummy Data
		int k=2;
		intb[] = new int[] {4,5};

		System.out.println(k);
		System.out.println(join_string(Arrays.toString(b)));

	}
	public static String join_string(String inp) {
		return inp.replace("[","").replace("]","").replace(",","");
	}
}

File: public/phase-2-20160620022203/C/elegance-of-a-sequence.c
// Name : Elegance of a Sequence
#include <stdio.h>
int main() {
	// input:
	// n : length of sequence of a
	// a : sequence a
	int n, a[105], k=2, b[105]={4,5}, index;
	scanf("%d", &n);
	for(index = 0; index< n; index++) {
		scanf("%d", &a[index]);
	}


	// write your code here
	// store your results in `k`,`b`

	// output
	printf("%d\n", k);
	for(index = 0; index< k; index++) {
		printf("%d ",b[index]);
	}
	return 0;
}